spec/models/work_spec.rb

```diff
@@ -330,16 +330,18 @@
     let(:draft_work) do
       work = Work.new(collection: collection, metadata: FactoryBot.build(:resource).to_json)
       work.draft!(user)
+      work = Work.find(work.id)
```

Comment:
Forces to make sure we use the value in the DB. I thought work.reload will do this but that didn't work.